scripts/templates_create_dagster_package/tox.ini.tmpl

```diff
@@ -0,0 +1,20 @@
+[tox]
+envlist = py{310,39,38,37}-{unix,windows}
+
+[testenv]
+usedevelop = true
+extras =
+  test
+setenv =
+  VIRTUALENV_PIP=21.3.1
+  !windows: COVERAGE_ARGS = --cov=dagster --cov-append --cov-report term:skip-covered --cov-report html --cov-report xml
+  windows: COVERAGE_ARGS =
+passenv = CI_* COVERALLS_REPO_TOKEN BUILDKITE*
```

Comment:
add TODO to fill out additional env var requirements and add them to buildkite/dagster-buildkite/dagster_buildkite/steps/packages.py